### ISSUE TITLE
Epic: Ongoing manual accessibility review of components entering Beta/RC

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#21984]
Components in the initial RC completed their manual a11y review, tracked in #19779.

This epic tracks manually accessibility testing for components entering beta/RC going forward:

- [x] Input: #21174 
- [x] Tabs: #21594
- [x] Slider: #21235
- [x] #24275
- [x] Switch: #21986
- [x] Checkbox: #22276

**Comment by GeoffCoxMSFT** [comment]
PM should decide how epics like this are tracked in the new unified board world.  This issue can also be tracked in the spreadsheet since we now have a solid template for any component.

**Comment by paulgildea** [comment]
This one specifically should be baked into the release criteria for any new component being brought into the library. Does the new component template include this as a checklist task that needs to be completed before release?

**Comment by tudorpopams** [comment]
@smhigley can you please check the Card issue that I just created and update it to use your template?
